### PR TITLE
QCOW support

### PR DESCRIPTION
**Pull request by uckelman-sf**
This PR add optional support for QCOW images, using libqcow.

### CODE REVIEW

File: tsk/img/img_open.cpp [hunk 1]
```diff
@@ -266,6 +290,24 @@ tsk_img_open(int num_img,
         break;
 #endif
 
+#if HAVE_LIBQCOW
+    case TSK_IMG_TYPE_QCOW_QCOW:
+        img_info = qcow_open(num_img, images, a_ssize);
+        break;
+#endif
+
+#if HAVE_LIBVHDI
```
Comment (cmueller-tp) [inline tsk/img/img_open.cpp:299]: LIBVHDI and LIBVMDK cases are already in line 281 and 287

**Review by cmueller-tp — COMMENTED** [review]
didn't do a full review, just realized while scrolling through.

File: tsk/img/img_open.cpp (hunk 1 above)
Comment (cmueller-tp) [inline tsk/img/img_open.cpp:299]: LIBVHDI and LIBVMDK cases are already handled above

Comment (uckelman-sf) [reply tsk/img/img_open.cpp:299]: Sorry, can you be more specific? I don't see what you're referring to.

**Review by cmueller-tp — APPROVED** [review]

**Comment by uckelman-sf** [comment]
Rebased over current develop.

**Comment by simsong** [comment]
@uckelman-sf  - I'm not sure how to resolve the conflict on this. Can you help?

**Comment by uckelman-sf** [comment]
@simsong This PR depends on some functionality in #2489. PR #2489 needs to be addressed first.

**Comment by simsong** [comment]
Great. As a reminder, this is still a draft.

**Comment by uckelman-sf** [comment]
@simsong It's still a draft because it cannot be merged before #2489. Will you have a look at that one, please?

**Comment by simsong** [comment]
> @simsong It's still a draft because it cannot be merged before #2489. Will you have a look at that one, please?

done.

**Comment by uckelman-sf** [comment]
@simsong Ready for review.

**Comment by simsong** [comment]
I fixed the conflicts between this and AFF4.

**Comment by uckelman-sf** [comment]
Fixed conflicts.

**Comment by simsong** [comment]
I can't figure out why `tsk/tsk_config.h.in` was included in the git repo, since its a file generated by autoconf.

**Pull request merged**